data-serving/scripts/setup-db/src/index.ts

```diff
@@ -43,6 +48,14 @@ const setupDatabase = async ({
 
         const collection = await database.getCollection(collectionName);
 
+        print('Dropping indexes👇');
```

Comment:
love the emojis, definitely elevating the vibe

Reply:
👍